substratevm/src/com.oracle.svm.core.genscavenge/src/com/oracle/svm/core/genscavenge/Space.java

```diff
@@ -386,66 +311,41 @@ void appendAlignedHeapChunk(AlignedHeapChunk.AlignedHeader aChunk) {
         }
     }
 
-    /**
-     * Append the argument AlignedHeapChunk to the doubly-linked list of AlignedHeapChunks. This
```

Comment:
I removed these comments because they are on private methods with an `@Uninterruptible` annotation and `Uninterruptibly` name suffix and their default-visible callers already contain checks and comments to that extent.